Fixes the error:

  ../src/util/u_queue.c:187:7: error: implicit declaration of function
  'timespec_get' is invalid in C99
  [-Werror,-Wimplicit-function-declaration]

        timespec_get(&ts, TIME_UTC);

diff --git a/src/util/u_queue.c b/src/util/u_queue.c
index b456871290d..963aa1c88d7 100644
--- a/src/util/u_queue.c
+++ b/src/util/u_queue.c
@@ -39,3 +39,4 @@
 #include <sys/syscall.h>
 #endif

+#include <time.h>

 /* Define 256MB */
 #define S_256MB (256 * 1024 * 1024)
